### PR TITLE
Add intra-cluster test to subctl latency benchmark

### PR DESCRIPTION
**Pull request by maayanf24**
Fixes issue #706

Signed-off-by: Maayan Friedman <maafried@redhat.com>

### CODE REVIEW

File: pkg/subctl/cmd/benchmark.go [hunk 1]
```diff
@@ -18,30 +20,38 @@ var (
...
 	}
 )
 
 func init() {
+	benchmarkLatenchCmd.PersistentFlags().BoolVar(&intraCluster, "intra-cluster-baseline", false,
+		"Run benchmark latency test with intra-cluster pods")
+
+	benchmarkThroughputCmd.PersistentFlags().BoolVar(&intraCluster, "intra-cluster-baseline", false,
+		"Run benchmark latency test with intra-cluster pods")
+
```
Comment (skitt) [inline pkg/subctl/cmd/benchmark.go:44]: I’m wondering if we need a separate flag for this, or if we can use the number of kubeconfigs given to choose... What do you think?

Comment (maayanf24) [reply pkg/subctl/cmd/benchmark.go:44]: Having the flag is more explicit.
Without the flag, if a user forget to add second kubeconfig the result will be another type of test.

Comment (skitt) [reply pkg/subctl/cmd/benchmark.go:44]: Right, more explicit is better since the failure scenario isn’t obvious.

**Review by skitt — APPROVED** [review]

File: pkg/subctl/cmd/benchmark.go [hunk 2]
```diff
@@ -18,30 +20,38 @@ var (
...
 		Args: func(cmd *cobra.Command, args []string) error {
-			return checkThroughputArguments(args)
+			return checkThroughputArguments(args, intraCluster)
 		},
 		Run: testLatency,
 	}
 )
 
 func init() {
+	benchmarkLatenchCmd.PersistentFlags().BoolVar(&intraCluster, "intra-cluster-baseline", false,
```
Comment (sridhargaddam) [inline pkg/subctl/cmd/benchmark.go:39]:
```suggestion
	benchmarkLatencyCmd.PersistentFlags().BoolVar(&intraCluster, "intra-cluster-baseline", false,
```

**Review by sridhargaddam — COMMENTED** [review]
small nit, otherwise LGTM.

**Review by sridhargaddam — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**